server/index.js

```
@@ -17,6 +19,11 @@ app.listen(port, (err) => {
   return console.log(`listening at port ${port}`);
 });
 
-app.get('/stocks', (req, res) => {
-  
-});
\ No newline at end of file
+app.get('/stocks/:companyId', (req, res) => {
+  db.getStockPrices(req.params.companyId, (err, data) => {
+    if (err) {
```

maybe return error instead of console logging error to stay consistent

---

Got it. Thanks for catching that!